### DISCUSSION TITLE
Cocaine + Alcohol

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#74] [Drug Interactions]
Just adding some sources for this combination. It's quite a popular one so I think having some sources highlighting the risks might be helpful.

I would also update the text to "The combination of cocaine and alcohol produces cocaethylene as a metabolite, which may be more cardiotoxic than cocaine or alcohol alone. Some studies indicate that cocaine + alcohol is more dangerous than either drug on its own. Use caution if combining these substances."

Sources:
McCance-Katz, E. F., Kosten, T. R., & Jatlow, P. (1998). Concurrent use of cocaine and alcohol is more potent and potentially more toxic than use of either alone—A multiple-dose study. Biological Psychiatry, 44(4), 250–259. https://doi.org/10.1016/S0006-3223(97)00426-5

Pergolizzi, J., Breve, F., Magnusson, P., LeQuang, J. A. K., & Varrassi, G. (2022). Cocaethylene: When cocaine and alcohol are taken together. Cureus, 14(2), e22498. https://doi.org/10.7759/cureus.22498

Shastry, S., Manoochehri, O., Richardson, L. D., & Manini, A. F. (2023). Cocaethylene cardiotoxicity in emergency department patients with acute drug overdose. Academic Emergency Medicine, 30(2), 82–88. https://doi.org/10.1111/acem.14584

Wiener, S. E., Sutijono, D., Moon, C. H., Subramanian, R. A., Calaycay, J., Rushbrook, J. I., & Zehtabchi, S. (2010). Patients with detectable cocaethylene are more likely to require intensive care unit admission after trauma. The American Journal of Emergency Medicine, 28(9), 1051–1055. https://doi.org/10.1016/j.ajem.2009.06.014

**Comment by utaninja** [comment]
The current combination says the following:
"Drinking on stimulants is risky because the sedative effects of the alcohol are reduced, and these are what the body uses to gauge drunkenness. This typically leads to excessive drinking with greatly reduced inhibitions, high risk of liver damage and increased dehydration. They will also allow you to drink past a point where you might normally pass out, increasing the risk. If you do decide to do this then you should set a limit of how much you will drink each hour and stick to it, bearing in mind that you will feel the alcohol less. Cocaine is potentiated somewhat by alcohol because of the formation of cocaethylene."

I like what you have written but also feel that we should mention the concern of canceling out effects and increased inhibition.

**Reply by zelixir25** [reply]
maybe something like:

Studies indicate that this combination is more dangerous than either drug on its own. Because cocaine can decrease some of the intoxicating effects of alcohol, some people may drink more than they otherwise would, leading to greater behavioral and physical health risks. Moreover, there is still some debate about cocaethylene, a metabolite produced when combining alcohol and cocaine, with some researchers positing that it is more cardiotoxic than cocaine on its own.

**Reply by utaninja** [reply]
Looks good! Pushing this

**Comment by utaninja** [comment]
@zelixir25 on discord:
"one combo i would want to talk about the classification for is cocaine + alcohol. the research seems fairly inconclusive about cocaethylene's cardiotoxicity. some studies say it's more cardiotoxic while others don't. given the disagreement i would honestly push for reclassifying it as caution rather than unsafe.
the risk, afaict,  is that both substances can inhibit some effects of the other which causes people to use more which results in greater risks rather than cocaethylene specifically causing more issues"

**Comment by utaninja** [comment]
Pushing to issue

**Comment by zelixir25** [comment]
Would like to revisit this. I thought I had brought up the possibility of reclassifying this as Caution instead of Unsafe but rereading this I actually did not. I think it would more accurately reflect the potential safety/risks of this combo to have it as Caution.

**Reply by utaninja** [reply]
It looks like this was last posted to have a caution flag but was mistakenly put as unsafe in the database. Making an issue now to have that changed. I completely agree.

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/332
